six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -123,28 +124,47 @@ Data* readNITF(const std::string& pathname,
     region.setBuffer(buffer.get() + offset);
     return reinterpret_cast<Data*>(reader.interleaved(region, 0));
 }
+%}
+
+%ignore six::sicd::cropSICD;
+%include <std_auto_ptr.i>
+%include <std_string.i>
+%include <std_vector.i>
+%template(VectorString) std::vector<std::string>;
+%auto_ptr(six::sicd::NoiseMesh);
+%auto_ptr(scene::ProjectionPolynomialFitter);
+%auto_ptr(six::sicd::ComplexData);
+%include "six/sicd/SICDMesh.h"
+%include "six/sicd/ComplexData.h"
+%include "six/sicd/Utilities.h"
+%rename (cropSICD) cropSICDWrap;
+
+%inline %{
+
 
+void cropSICDWrap(const std::string& inPathname,
```

Comment:
I don't quite follow... these are named as `cropSICDWrap()` and then you have a `%rename` to change `cropSICD()` to `cropSICDWrap()`.  Don't we want to go the other direction?  That is, don't we want the function to be called `cropSICD()` when we call in Python?

Reply:
The rename changes `cropSICDWrap` to `cropSICD` for the Python level.

Reply:
Ahhhhh - I get it.  OK cool.